setup.py

```diff
@@ -19,17 +19,17 @@
 PYTHON_REQUIRES = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*"
 
 INSTALL_REQUIRES = [
-    'attrs>=17.4',
-    'plumbum>=1.6.6',
+    'attrs>=19.2',
```

Comment:
I'd restore this, unless we use features in 19.2.

Reply:
I should have been explicit here - checking the dependencies of `Particle` I noticed the upgrade there, which is this one. Hence, in practice, we will have to use 19.2 coming from the requirement in `Particle`. I would not go and change the dependency in `Particle` at this stage.

Reply:
OK, @henryiii ?

Reply:
That’s okay, then; though does DL use attrs? We can just inherit Particle’s if we don’t use it directly

Reply:
Yep, you use it in your classes related to AmpGen.